optaplanner-webexamples/pom.xml

```diff
@@ -23,7 +23,7 @@
   <url>https://www.optaplanner.org</url>
 
   <properties>
-    <version.org.wildfly>10.1.0.Final</version.org.wildfly>
+    <version.org.wildfly>11.0.0.Final</version.org.wildfly>
```

Comment:
Has it been tested that the optaplanner-webexamples still work? Especially their arquillain test?

We need to move this version.org.wildfly to kie-parent or ip-bom. But I am even more worried about the relationship with the arquillian versions - that stuff is really brittle.